bugbug/db.py

```diff
@@ -80,3 +90,27 @@ def delete(path, match):
 
     os.unlink(path)
     os.rename(f'{path}_new', path)
+
+
+def is_updated_ver(db_path):
```

Comment:
The naming is a bit confusing, I was assuming `is_updated_ver` meant 'Is the database up to date?', but instead it means 'Is the database out of date?'.
I'd either make this be 'Is the database up to date?' (returning True when the download shouldn't be done and False when it should), or change its name.